Xamarin.PropertyEditing.Mac/Controls/FilePathEditorControl.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using AppKit;
+using CoreGraphics;
+using Xamarin.PropertyEditing.Mac.Resources;
+using System.IO;
+using Xamarin.PropertyEditing.Common;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal class FilePathEditorControl : BasePathEditorControl<FilePath>
+	{
+		public FilePathEditorControl (IHostResourceProvider hostResource)
+		: base (hostResource)
+		{
+			this.currentTextField.ToolTip = this.currentTextField.PlaceholderString = string.Format (LocalizationResources.ChooseFileOrDirectory, LocalizationResources.File);
+			this.panel.CanChooseFiles = true;
+			this.panel.CanChooseDirectories = false;
+			this.revealPathButton.ToolTip = string.Format (LocalizationResources.RevealFileOrDirectory, LocalizationResources.File);

```

Comment:
Does Voice Over read these if there's no accessibility titles? Perhaps we should have titles.

Reply:
In NSButtons the accessibility.title is by default the title of the button AFAIK

Reply:
Yep. But we can include it if you want.

![filepath](https://user-images.githubusercontent.com/1587480/52675734-06c0b280-2f28-11e9-8488-601b3be62548.gif)